src/library/scala/collection/Seq.scala

```diff
@@ -181,7 +181,8 @@ trait SeqOps[+A, +CC[_], +C] extends Any
 
   // Make `concat` an alias for `appendedAll` so that it benefits from performance
   // overrides of this method
-  @`inline` final override def concat[B >: A](suffix: Iterable[B]): CC[B] = appendedAll(suffix)
+  // TODO https://github.com/scala/bug/issues/10853 Uncomment final
```

Comment:
Let's make it `@deprecatedOverriding` like here https://github.com/scala/scala/pull/6843/files

Reply:
Done.

Reply:
Thanks!